cards/multilang-faq-accordion/template.hbs

```diff
@@ -1,4 +1,4 @@
-<div class="HitchhikerFaqAccordion yxt-{{cardName}} js-HitchhikerFaqAccordion{{#if card.isExpanded}} HitchhikerFaqAccordion--expanded{{/if}}"
+<div class="HitchhikerFaqAccordion yxt-{{cardName}} js-HitchhikerFaqAccordion{{#if card.isExpanded}} HitchhikerFaqAccordion--expanded{{/if}}  js-HitchhikerFaqAccordion{{#if card.isExpanded}} HitchhikerFaqAccordion--expanded{{/if}}  HitchhikerFaqAccordion--stayExpanded"
```

Comment:
do we need `HitchhikerFaqAccordion--stayExpanded` to be in the template? in the case that card.isExpanded is false, we don't want that class right? this seems like something that should be added in JS code when you hit the feedback button

Reply:
Yes, it can be removed

Reply:
Actually, it can't be removed because the component is mounted again when feedback is submitted. Instead, I can remove the code adding the class on a feedback button click